src/Compilers/Core/Portable/WellKnownMembers.cs

```diff
@@ -4310,6 +4310,15 @@ static WellKnownMembers()
                         1,
                         (byte)SignatureTypeCode.GenericTypeParameter, 0,
 
+                // System_Collections_Immutable_ImmutableArray_T__Empty
+                (byte)(MemberFlags.Field | MemberFlags.Static),                                                                                    // Flags
+                (byte)WellKnownType.System_Collections_Immutable_ImmutableArray_T,                                          // DeclaringTypeId
```

Comment:
```suggestion
                // System_Collections_Immutable_ImmutableArray_T__Empty
                (byte)(MemberFlags.Field | MemberFlags.Static),                                                                                  // Flags
                (byte)WellKnownType.System_Collections_Immutable_ImmutableArray_T,                                          // DeclaringTypeId
```

Alignment appears off in GitHub diff.